web-console/5-components/wizard-results.md

```diff
@@ -0,0 +1,66 @@
+# Wizard Results
+
+The results step for the "add to projects" wizards should follow a consistent format while supporting the unique needs of the various methods by which users may add resources to their projects. The experience should be similar, even if the types of information to be displayed is different.
+
+## Design
+
+![results steps callouts](img/results-step-callouts.png)
+
+1. A status icon followed by the status. The status should call out the resource being created by name and if the action is initiated from outside of a project, the project name should be included as well.
+  - Status icons
+    - `fa-clock-o` for provisioning
+    - `pficon-ok` for success
+    - `pficon-error-circle-o` for failure
+1. Additional details about the status, such as error messages, additional actions to be performed or estimates for how long the operation may take.
+1. An action to take users to the overview of the project that the operation affects.
+1. Other information specific to the type of operation, for example, template messages, applied parameters, documentation, etc.
+
+
+## Implementation details
+
+### Provisioned Services Without Bindings
+![results from a provisioned service with no binding](img/results-provisioned-service-no-binding.png)
+1. When a provision is in progress, the status should read "X is being provisioned in Y Project" where X is the Service Class Display Name. Other statuses should read similarly: "...has been provisioned successfully..." or "...failed to provision..."
+1. The UI should give the user an indication of how long the provision may take. If no specific message is provided by the broker, the default should set the user's expectation, but need not be overly specific, for example "This may take several minutes."
+  - There is an [issue](https://github.com/openservicebrokerapi/servicebroker/issues/315) open to the Open Service Broker to allow brokers to return a message for async operations before polling. 
+1. Links to documentation, support and a dashboard should be provided if those links exist (services may not provide all three items).
+
+![results provisioned service fail](img/results-provisioned-service-fail.png)
+
+If a service failed to provision, users should be given:
+1. The relevant error message.
+1. Links to service documentation and support information.
+1. TODO The ability to delete failed services.
+
+### Provisioned Services With Bindings
+
+![results from a provisioned service with binding 1](img/results-provision-bind-1.png)
+- Text underneath the provision status informs the user that bindings are only created after the service has been provisioned.
+
+![results from a provisioned service with binding 2](img/results-provision-bind-2.png)
+- Once the service has been provisioned successfully, the status of the binding is updated underneath the completion message for the service.
+
+![results from a provisioned service with binding 3](img/results-provision-bind-3.png)
```

Comment:
This pertains to all screens but is particularly evident here. Because of the extra large icons, the alignment seems off when there are more than 2 lines of text. Currently icons are being centered on the content which looks particularly odd when it comes to smaller screens.

Other areas in PF where we use icons next to text such as inline notifications or toasts, we always top align the icon with the first line of text. Seeing that the first line of success or failure is larger, I would think the icon could line up accordingly if we made it slightly smaller. If I'm alone on this thinking, feel free to ignore!